example-app/default.local.gradle

```diff
@@ -11,7 +11,6 @@ android {
             buildConfigField "String", "MERCHANT_SERVER_URL", "\"<YOUR_SERVER_URL>\""
             buildConfigField "String", "API_KEY_HEADER_NAME", "\"<API_KEY_HEADER>\""
             buildConfigField "String", "CHECKOUT_API_KEY", "\"<YOUR_API_KEY>\""
-            buildConfigField "String", "PUBLIC_KEY", "\"<PUBLIC_KEY>\""
```

Comment:
@caiofaustino same here

Reply:
Yep, now we should have a default for the client key right? `<CLIENT_KEY>`

Reply:
nvm we already have that